src/pages/daoMember/daoMember.tsx

```diff
@@ -223,10 +235,8 @@ export const DaoMember: React.FC = () => {
         }
       />
       <div className="flex flex-col gap-16 px-4 md:flex-row md:px-0">
-        <div className="flex grow flex-col gap-10">
-          <UserProposalList proposals={memberCreatedProposals} />
-        </div>
-        <div className="flex w-full grow md:max-w-[400px]" />
+        <UserProposalList proposals={memberCreatedProposals} />
+        <MembershipDAOList daos={daoMemberList as MemberDAOsType} />
```

Comment:
I wouldn't cast the `daoMemberList` to `MemberDAOsType` here as it might be undefined in case the call fails.
```suggestion
        <MembershipDAOList daos={daoMemberList} />
```
Instead, I would update the `MembershipDAOList` component prop to handle an undefined list:
```
export interface IMembershipDAOListProps {
  daos?: MemberDAOsType;
}
```